test/common/http/alternate_protocols_cache_impl_test.cc

```diff
@@ -116,6 +130,47 @@ TEST_F(AlternateProtocolsCacheImplTest, FindAlternativesAfterTruncation) {
   EXPECT_EQ(expected_protocols, protocols.ref());
 }
 
+TEST_F(AlternateProtocolsCacheImplTest, ToAndFromString) {
+  auto testAltSvc = [&](const std::string& original_alt_svc,
+                        const std::string& expected_alt_svc) -> void {
+    auto protocols = AlternateProtocolsCacheImpl::protocolsFromString(original_alt_svc, simTime());
```

Comment:
nit: Can you include the type for `protocols` and `protocol` down on line 140 (and `protocol2` on line 149)?